pysat/instruments/omni_hro.py

```diff
@@ -181,8 +181,8 @@ def time_shift_to_magnetic_poles(inst):
     for i, time in enumerate(time_x_offset):
         new_index.append(inst.data.index[i] + time)
     inst.data.index = new_index
-    inst.data = inst.data.sort_index()    
-    
+    inst.data = inst.data.sort_index()
+
     return
 
 def download(date_array, tag, sat_id='', data_path=None, user=None,
```

Comment:
This should probably be replaced with the CDAWeb call. I think I just missed converting OMNI when I transitioned the other instruments over.

Reply:
I had tried that initially, but was having issues with the `fake_daily_files_from_monthly` keyword.  Any suggestions?

Reply:
updated using the CDAweb call